php/utils.php

```diff
@@ -1100,7 +1100,9 @@ function glob_brace( $pattern, $dummy_flags = null ) {
 function get_suggestion( $target, array $options, $threshold = 2 ) {
 
 	$suggestion_map = array(
+		'add' => 'create',
```

Comment:
With suggestions like this one (and the `'trash' => 'delete'` one as well), we should first make sure that only valid options are presented as suggestions (see #4578).

There's some overlap for these terms across the different commands, so we want to make sure they are only shown when they are actually relevant.

Also, once #4578 got in, we would probably want to provide these options in both directions.

Reply:
Good points

Reply:
@johnbillion #4578 was now fixed.

Could you pull in latest master, and add the two suggestions that might make sense in both directions in swapped direction as well?

Reply:
I cannot push to this branch, I'll merge and then create an additional PR instead.